src/httpclient.cc

```diff
@@ -46,6 +47,11 @@ HttpClient::HttpClient() {
   if (loggerGetSeverity() <= LVL_debug) {
     curl_easy_setopt(curl, CURLOPT_VERBOSE, 1L);
   }
+
+  struct curl_slist* h = curl_slist_append(headers, "Accept: */*");
+  h = curl_slist_append(headers, "Content-Type: application/json");
+  curl_easy_setopt(curl, CURLOPT_HTTPHEADER, h);
+  curl_slist_free_all(headers);
```

Comment:
I think will free `headers` too soon. From https://curl.haxx.se/libcurl/c/CURLOPT_HTTPHEADER.html:

"When this option is passed to curl_easy_setopt, libcurl will not copy the entire list so you must keep it around until you no longer use this handle for a transfer before you call curl_slist_free_all on the list. "

We call curl_slist_free_all in the destructor, so I think this isn't needed.